src/main/java/core/basesyntax/UserService.java

```diff
@@ -1,8 +1,15 @@
 package core.basesyntax;
 
-public class UserService {
+import core.basesyntax.exception.UserNotFoundException;
 
+public class UserService {
     public int getUserScore(String[] records, String email) {
-        return 0;
+        for (String user : records) {
+            String [] emailAndScoreForUser = user.split(":");
+            if (emailAndScoreForUser.length == 2 && emailAndScoreForUser[0].equals(email)) {
```

Comment:
```suggestion
            if (emailAndScoreForUser[0].equals(email)) {
```
no need to check length